lib/templates/deploy.batch.aspnet.core.template

```diff
@@ -0,0 +1,39 @@
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+:: Deployment
+:: ----------
+
+echo Handling ASP.NET Core Web Application deployment.
+
+:: Remove wwwroot if deploying to default location
+IF "%DEPLOYMENT_TARGET%" == "%WEBROOT_PATH%" (
+    FOR /F %%i IN ("%DEPLOYMENT_TARGET%") DO IF "%%~nxi"=="wwwroot" (
+    SET DEPLOYMENT_TARGET=%%~dpi
+    )
+)
+
+:: Remove trailing slash if present
+IF "%DEPLOYMENT_TARGET:~-1%"=="\" (
+    SET DEPLOYMENT_TARGET=%DEPLOYMENT_TARGET:~0,-1%
+)
+
+:: We don't need the XML docs
+set NUGET_XMLDOC_MODE=skip
+
+:: Make the NuGet packages go in a persisted folder instead of %USERPROFILE%, which on Azure goes in Temp space
+set NUGET_PACKAGES=%HOME%\.nuget
```

Comment:
We should ideally make this Azure conditional. I guess as it stands, it won't work locally probably for various reasons. I wouldn't block on it for now, but we should make it a point to make it run locally in a subsequent change.